sdks/python/apache_beam/typehints/native_type_compatibility.py

```diff
@@ -85,7 +85,7 @@ def _safe_issubclass(derived, parent):
   """
   try:
     return issubclass(derived, parent)
-  except TypeError:
+  except (TypeError, AttributeError):
```

Comment:
Will issubclass actually raise an AttributeError? Curious what causes this

Reply:
I'm pretty sure I did see this. (Maybe it was missing `__mro__` or something?)